build.ps1

```diff
@@ -69,6 +69,10 @@ bazel build //... `
 bazel shutdown
 
 if ($env:SKIP_TESTS -ceq "False") {
+    # Generate mapping from shortened scala-test names on Windows to long names on Linux and MacOS.
+    ./ci/remap-scala-test-short-names.ps1 `
+      | Out-File -Encoding UTF8 -NoNewline scala-test-suite-name-map.json
```

Comment:
This step takes about 4.5 seconds on a Windows ad-hoc machine. The resulting file is about 70kB large.